From 5f25c4bb92d3fd8a8da377a156c2d49c34a2431d Mon Sep 17 00:00:00 2001
Message-Id: <5f25c4bb92d3fd8a8da377a156c2d49c34a2431d@dist-git>
From: Jiri Denemark <jdenemar@redhat.com>
Date: Tue, 5 Jul 2016 10:07:24 +0200
Subject: [PATCH] qemu: Properly reset spiceMigration flag

Otherwise migration during which we didn't send client_migrate_info QMP
command will get stuck waiting for SPICE migration to finish if libvirtd
sent the QMP command in a previous migration attempt.

Broken by bd7c8a69.

https://bugzilla.redhat.com/show_bug.cgi?id=1151723

Signed-off-by: Jiri Denemark <jdenemar@redhat.com>
(cherry picked from commit a16ea1a0f3e6b9eb8be4be7a664af76e47bbceba)
Signed-off-by: Jiri Denemark <jdenemar@redhat.com>
---
 src/qemu/qemu_domain.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/qemu/qemu_domain.c b/src/qemu/qemu_domain.c
index 0edcc04..4aef7ab 100644
--- a/src/qemu/qemu_domain.c
+++ b/src/qemu/qemu_domain.c
@@ -217,6 +217,7 @@ qemuDomainObjResetAsyncJob(qemuDomainObjPrivatePtr priv)
     job->mask = QEMU_JOB_DEFAULT_MASK;
     job->dump_memory_only = false;
     job->abortJob = false;
+    job->spiceMigration = false;
     job->spiceMigrated = false;
     job->postcopyEnabled = false;
     VIR_FREE(job->current);
-- 
2.9.0

